src/hyperion/device_setup_plans/setup_oav.py

```diff
@@ -163,3 +190,21 @@ def wait_for_tip_to_be_found(mxsc: MXSC) -> Generator[Msg, None, Tuple[int, int]
             f"No pin found after {pin_tip.validity_timeout.get()} seconds"
         )
     return found_tip
+
+
+def wait_for_tip_to_be_found_ophyd(
+    ophyd_pin_tip_detection: PinTipDetection,
+) -> Generator[Msg, None, Tuple[int, int]]:
+    found_tip = yield from bps.rd(ophyd_pin_tip_detection)
+
+    LOGGER.info("Pin tip not found, waiting a second and trying again")
+
+    if found_tip == ophyd_pin_tip_detection.INVALID_POSITION:
```

Comment:
Do I understand correctly that this will be adressed in https://github.com/DiamondLightSource/hyperion/issues/1069?

Reply:
Yep